src/main/resources/static/js/createSurvey.js

```diff
@@ -169,3 +217,22 @@ function addRangeQuestionToSurvey(questionPrompt, inputType, questionMin, questi
     })
   })
 }
+
+function addMCQuestionToSurvey(questionPrompt, inputType, options){
+ return new Promise((resolve, reject) => {
+    apiUrl = window.location.origin + '/api/question'
+    $.ajax({
+      url: apiUrl,
+      type: 'POST',
+      contentType: "application/json",
+      data: JSON.stringify({ survey: surveyHref, type: inputType, prompt: questionPrompt , options: options}),
```

Comment:
options should be called choices to match the DB entity

Reply:
Sure I'll make that change